### PR TITLE
Clarify `@export_enum` docs

### PR DESCRIPTION
**Pull request by dalexeev**
According to godotengine/godot#72305.

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_exports.rst [hunk 1]
```diff
@@ -298,14 +298,22 @@ of the selected option (i.e. ``0``, ``1``,  or ``2``).
 
 ::
 
-    @export_enum("Warrior", "Magician", "Thief") var character_class
+    @export_enum("Warrior", "Magician", "Thief") var character_class: int
+
+You can specify other values using a colon::
+
+    @export_enum("Slow:30", "Average:60", "Very Fast:200") var character_speed: int
```
Comment (raulsntos) [inline tutorials/scripting/gdscript/gdscript_exports.rst:305]: This syntax is also available for `@export_flags`.

File: tutorials/scripting/gdscript/gdscript_exports.rst [hunk 2]
```diff
@@ -266,6 +266,15 @@ has value 1, ``Water`` has value 2, ``Earth`` has value 4 and ``Wind``
 corresponds to value 8. Usually, constants should be defined accordingly (e.g.
 ``const ELEMENT_WIND = 8`` and so on).
 
+You can add explicit values using a colon::
+
+    @export_flags("Self:4", "Allies:8", "Foes:16") var spell_targets = 0
+
+You can also combine several flags::
```
Comment (YuriSizov) [inline tutorials/scripting/gdscript/gdscript_exports.rst:273]:
```suggestion
Only power of 2 values are valid as bit flags options. The lowest allowed value
is 1, as 0 means that nothing is selected. You can also add options that are a
combination of other flags::
```

File: tutorials/scripting/gdscript/gdscript_exports.rst [hunk 3]
```diff
@@ -281,9 +290,9 @@ If in doubt, use boolean variables instead.
 Exporting enums
 ---------------
 
-Properties can be exported with a type hint referencing an enum to limit their values
-to the values of the enumeration. The editor will create a widget in the Inspector, enumerating
-the following as THING_1, THING_2, ANOTHER_THING. The value will be stored as an integer.
+Properties can be exported with a type hint referencing an enum. The editor will
+create a widget in the Inspector, enumerating the following as "Thing 1", "Thing 2",
+"Another Thing". The value will be stored as an integer.
```
Comment (YuriSizov) [inline tutorials/scripting/gdscript/gdscript_exports.rst:295]:
```suggestion
Properties can be exported with a type hint referencing an enum to limit their values
to the values of the enumeration. The editor will create a widget in the Inspector, enumerating
the following as "Thing 1", "Thing 2", "Another Thing". The value will be stored as an integer.
```

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**